Challange-1-iOS/Challange-1-iOS/Persistence/AvatarPersistence.swift

```diff
@@ -3,65 +3,34 @@ import CoreData
 import RxSwift
 
 class AvatarPersistence {
-    var avatarsPersistenceList: [NSManagedObject] = []
     private let persistentContainer: NSPersistentContainer
 
     init(persistentContainer: NSPersistentContainer) {
         self.persistentContainer = persistentContainer
     }
 
-    func saveAvatar(currentAvatar: Avatar) {
-
-        DispatchQueue.main.async {
-
-            // 1
+    func saveAvatar(currentAvatar: Avatar) -> Completable {
```

Comment:
the delete method name is `delete(avatar ...)`, to keep it coherent this could be named as `save(avatar ...)`